api/src/main/java/io/strimzi/api/kafka/model/listener/arraylistener/GenericKafkaListenerConfiguration.java

```diff
@@ -229,6 +230,17 @@ public void setIpFamilies(List<IpFamily> ipFamilies) {
         this.ipFamilies = ipFamilies;
     }
 
+    @Description("Whether to create the bootstrap service or not." +
```

Comment:
You need to add a space after the `.`. Otherwise in the docs this is then rendered as `Whether to create the bootstrap service or not.This field can be used with the loadBalancer type listener.` without the space between the sentences.